tools/encyclopedia/encyclopedia_searchtolib.xml

```diff
@@ -0,0 +1,177 @@
+<tool id="encyclopedia_searchtolib" name="SearchToLib" version="@VERSION@.0">
+    <description>Build a Chromatogram Library from Data-Independent Acquisition (DIA) MS/MS Data</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive"><![CDATA[
+        @CMD_IMPORTS@
+        @LINK_SCAN_INPUTS@
+        @LINK_FASTA_INPUT@
+        @LINK_TARGET_FASTA@
+        @LINK_LIB_INPUT@
+        for SCAN_FILE in `ls -1 inputs/*`; do 
+        echo "\$SCAN_FILE" &&
```

Comment:
indentation? Can this be shared with the other tool?

Reply:
Moved to macros.